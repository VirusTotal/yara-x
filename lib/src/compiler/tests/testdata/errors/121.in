rule test {
  condition:
    "foo" matches /foo/x
}