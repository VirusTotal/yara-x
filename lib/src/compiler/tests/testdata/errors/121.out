error: invalid regexp modifier `x`
 --> line:3:24
  |
3 |     "foo" matches /foo/x
  |                        ^ invalid modifier
  |